### ISSUE TITLE
Plugin won't load in Docker, but works fine with Grafana installed as Debian package

### ISSUE THREAD

**Issue by rlueckl** [grafana/grafana#27924]
**What happened**: I'm getting the following message when trying to add the [Discrete](https://grafana.com/grafana/plugins/natel-discrete-panel) plugin to my Grafana Docker installation:

```
t=2020-09-30T08:17:38+0000 lvl=warn msg="Plugin missing module.js" logger=plugins name=Discrete warning="Missing module.js, If you loaded this plugin from git, make sure to compile it." path=/etc/grafana/plugins/natel-discrete-panel/src/module.js
```

**What you expected to happen**: The plugin is loaded without any problems.

**How to reproduce it (as minimally and precisely as possible)**: [Download](https://grafana.com/api/plugins/natel-discrete-panel/versions/0.1.0/download) the plugin as Zip, extract it and add to the Docker image:

```
FROM grafana/grafana:7.1.5
# I have multiple plugins in the "plugins" directory and add all of them at once
ADD plugins /etc/grafana/plugins/
WORKDIR /usr/share/grafana
ENTRYPOINT [ "/run.sh" ]
```

Set path in `grafana.ini`:

```
    [plugin.discrete]
    path = /etc/grafana/plugins/natel-discrete-panel
```

When trying to start the image I get the above mentioned warning in the log.

**Anything else we need to know?**: I also have Grafana installed from a .deb file (on Debian Buster) and extracted the same plugin to `/var/lib/grafana/plugins`. Grafana starts without any problems and the plugin works as expected.

The file `src/module.js` *doesn't exist in either installation*! The .deb-based installation just doesn't seem to care.

Why is the behavior of Grafana different when the Docker image is used?

**Environment**:
- Grafana version: 7.1.5 (both, Docker and .deb)
- Data source type & version: -
- OS Grafana is installed on: Debian Buster (.deb version)
- User OS & Browser: -
- Grafana plugins: "Discrete" by Natel Energy, version 0.1.0
- Others: -

**Comment by torkelo** [comment]
Must be something wrong when you prep and copy the plugin to the folder in the docker file. 

Better to copy the plugin to GF_PATHS_PLUGINS 

https://github.com/grafana/grafana/blob/master/packaging/docker/run.sh#L88

or use our preferred way to install plugins:
https://grafana.com/docs/grafana/latest/installation/docker/#install-plugins-in-the-docker-container

**Comment by rlueckl** [comment]
Thanks!

In the meantime I've found the problem, so for anybody who might stumble upon this:

The Zip from https://grafana.com/grafana/plugins/natel-discrete-panel/installation contains a `.gitignore` which has the `dist/` directory listed. Since I've added this plugin to my own Git project (the one which builds the Docker image) I accidentally added the `.gitignore` file too. This was recognized by my Git repo (even though it's in a sub-sub-directory), so the `dist/` directory wasn't added to my Git repo and thus wasn't added to the Docker image.

Probably that's why Grafana started to look for `module.js` in the `src/` directory...

Removing the `.gitignore` file and adding the `dist/` directory properly fixed the issue.

Unfortunately I can't use the preferred way to install plugins since I'm behind a company firewall and grafana-cli would need to use a proxy.
